scripts/set-tools-host-labels.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
+
+# Add the `host` label to all RStudio/JupyterLab resources
+
+
+set -e
```

Comment:
You're handling envvars not being set so won't suggest `set -u`.  Maybe `set -o pipefail` might be useful